spinalcordtoolbox/scripts/sct_extract_metric.py

```diff
@@ -369,10 +369,17 @@ def main(argv: Sequence[str]):
 
     # Combine individual labels for estimation
     if combine_labels:
-        # Add entry with internal ID value (99) which corresponds to combined labels
-        label_struc[99] = LabelStruc(id=labels_id_user, name=','.join([str(i) for i in labels_id_user]),
-                                     map_cluster=None)
-        labels_id_user = [99]
+        # Avoid case where user asked to combine a single label which is already a combination of 'native' labels
+        # Related to https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4467
+        if not (len(labels_id_user)==1 and labels_id_user[0] in combined_labels_ids):
```

Comment:
I don't think we necessarily need the `labels_id_user[0] in combined_labels_ids` part? Since, `-combine` shouldn't do anything if there is only 1 label (regardless of whether it's `Individ` or `Combined`).

Reply:
hum, I think you are correct